konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/build/extension/ProfileCommand.java

```diff
@@ -0,0 +1,509 @@
+                log.error("Successfully set '{}' profile as default.", profileName);
+            } catch (IOException e) {
+                log.error("Unable to set default profile", e);
+            }
+        }
+    }
+
+    public static Profile getDefaultProfile() {
+        try {
+            String profileName = FileUtils.readFileToString(defaultProfileNamePath, StandardCharsets.UTF_8);
+            return getProfile(profileName);
+        } catch (IOException e) {
+            log.error("Unable to get default profile", e);
+            return null;
+        }
+    }
+
+    private void createProfile() {
+        if(isProfileExists(profileName)) {
+            out.format("Profile with name %s already exists.%n", profileName);
+        } else {
+            saveProfile(profileName, fillProfileValues(new Profile()));
+        }
+    }
+
+    private void editProfile() {
+        if(isProfileExists(profileName)) {
+            if(profileName.equals("CPU") || profileName.equals("CUDA")) {
+                out.format("Cannot edit pre-set profiles with name 'CPU' or 'CUDA'.%n");
+                System.exit(1);
+            } else {
+                saveProfile(profileName, fillProfileValues(getProfile(profileName)));
+            }
+        } else {
+            out.format("No profile found with the name of %s%n", profileName);
+        }
+    }
+
+    private Profile fillProfileValues(@NonNull Profile profile) {
+        if(cpuArchitecture != null) {
+            profile.cpuArchitecture(cpuArchitecture);
+        }
+
+        if(operatingSystem != null) {
+            profile.operatingSystem(operatingSystem);
+        }
+        if(computeDevice != null) {
+            profile.computeDevice(computeDevice);
+        }
+
+        if(StringUtils.containsIgnoreCase(profile.computeDevice(), "cuda")) {
+            profile.cpuArchitecture("x86");
+        }
+
+        if(serverTypes != null && !serverTypes.isEmpty()) {
+            profile.serverTypes(serverTypes);
+        }
+        if(additionalDependencies != null && !additionalDependencies.isEmpty()) {
+            profile.additionalDependencies(additionalDependencies);
+        }
+
+        if(StringUtils.containsIgnoreCase(profile.computeDevice(), "cuda")) {
+            String cudaVersion = profile.computeDevice().split("_")[1].trim();
+            String cudaRedistPackage = null;
+            Pair<String, String> cudaInstall = findCudaInstall();
+            if(cudaInstall == null) {
+                switch (cudaVersion) {
+                    case "10.0":
+                        out.format("No CUDA install found and no available redist package for cuda version: '%s' found. " +
+                                "Make sure to install CUDA from: %s before starting a konduit server.%n", cudaVersion,
+                                "https://developer.nvidia.com/cuda-10.0-download-archive");
```

Comment:
We should probably explicitly mention the user passed CUDA 10.0 and that they could use CUDA 10.1 or 10.2 as an alternative.